aspnetcore/host-and-deploy/linux-nginx.md

```diff
@@ -279,6 +279,21 @@ For further filtering, time options such as `--since today`, `--until 1 hour ago
 sudo journalctl -fu kestrel-hellomvc.service --since "2016-10-18" --until "2016-10-18 04:00"
 ```
 
+## Data protection
+
+The [ASP.NET Core Data Protection stack](xref:security/data-protection/index) is used by several ASP.NET Core [middlewares](xref:fundamentals/middleware/index), including middleware used in authentication (for example, cookie middleware) and cross-site request forgery (CSRF) protections.. Even if Data Protection APIs aren't called by user code, data protection should be configured to create a persistent cryptographic [key store](xref:security/data-protection/implementation/key-management). If data protection isn't configured, the keys are held in memory and discarded when the app restarts.
```

Comment:
- There's an extra period after the 1st sentence. Also, I suggest shortening "including middleware used in authentication", as recommended above.
- Consider making this Data Protection section an include. It appears to be the same as the one used in the *linux-apache.md* file.

Reply:
I lean away from an include here. Since these are SxS sister topics, the passage is only used in these two topics, and they both map to me, I can keep them updated without the ceremony.

There's an extension opportunity here for VSC ... a *show me the include feature.* Hover on the include, and it shows the text and provides a quick link to open the file. I'd like using them more even in a simple case like this if such a thing existed.